### PR TITLE
docs: readme

### PR DESCRIPTION
**Pull request by mymphe**
Added Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,94 @@
...
+
 # state-mate
+
+state-mate is a simple automation tool that validates the protocol state against an input configuration. Run state-mate to verify deploy parameters, current state, access control and more.
+
+state-mate accepts a yaml file that includes contract addresses, view functions and their expected results. It calls each function and compares the output to the expected result.
+
+## ✨ Features
+
+- state verification,
```
Comment (arwer13) [inline README.md:15]:
```suggestion
- state (non-mutable functions result) verification,
- automatically validates that all functions covered,
```

File: README.md [hunk 2]
```diff
@@ -1 +1,94 @@
...
+l1:
+  rpcUrl: L1_MAINNET_RPC_URL # env variable
+  contracts:
+    myContract:
+      name: "myContract"
+      address: *myContract
+      checks:
+        # list of view functions and expected results
+        getMyParameter: *MY_PARAMETER
+        getFoo: *FOO
```
Comment (arwer13) [inline README.md:81]:
```suggestion
        getFoo: *FOO
      ozAcl:
        *DEFAULT_ADMIN_ROLE : [*adminMultisig]
```

File: README.md [hunk 3]
```diff
@@ -1 +1,94 @@
...
+  # List of parameters
+  - &MY_PARAMETER 42
+
+misc:
+  # Misc variables
+  - &FOO "foo"
+
+deployed:
+  # Contract addresses
+  - &myContract "0x0000000000000000000000000000000000000001"
```
Comment (arwer13) [inline README.md:66]:
```suggestion
  - &myContract "0x0000000000000000000000000000000000000001"
  - &adminMultisig "0x0000000000000000000000000000000000000002"
```

File: README.md [hunk 4]
```diff
@@ -1 +1,94 @@
...
+roles:
+  # ACL checks
+  - &DEFAULT_ADMIN_ROLE "0x0000000000000000000000000000000000000000000000000000000000000000"
+
+l1:
+  rpcUrl: L1_MAINNET_RPC_URL # env variable
+  contracts:
+    myContract:
+      name: "myContract"
+      address: *myContract
```
Comment (arwer13) [inline README.md:77]:
```suggestion
      address: *myContract
      implementation: "%implementation address%"
      proxyChecks:
        proxy__getAdmin: *adminMultisig
```

File: README.md [hunk 5]
```diff
@@ -1 +1,94 @@
...
+
+All requried ABIs must be located in the same directory as the config and placed under `abi` folder. See [configs](/configs/).
+
+## 🔧 Contributing
+
+Any contributions to this project are welcome. Please fork the repository and submit pull requests with detailed descriptions of your changes. Or you can submit an issue, bug report or feature request.
+
+## 📃 License
+
+This project is licensed under the MIT License. See the [LICENSE](/LICENSE) file for details.
```
Comment (arwer13) [inline README.md:94]:
```suggestion
This project is licensed under the MIT License. See the [LICENSE](/LICENSE) file for details.

```

**Comment by arwer13** [comment]
Good start, thank you! The first version helps much to start moving. Please accept the suggested changes if consider proper

**Review by arwer13 — APPROVED** [review]

**Comment by mymphe** [comment]
> It looks great, but I'm missing a link to examples (reference current configs, e.g.)

I've added a sample config and a link to configs dir in the ABI section. What do you suggest I add as well?

**Review by tamtamchik — APPROVED** [review]
🎉

File: README.md [hunk 6]
```diff
@@ -1 +1,102 @@
...
+- CI-friendly,
+- uses yaml config (env variables, variables, comments).
+
+## ⚡ Getting started
+
+### Requirements
+
+- git
+- Node.js >=20,
+- yarn 1.x
```
Comment (TheDZhon) [inline README.md:28]: Wow, shall we use modern yarn instead, given the up-to-date LTS node.js is required?

File: README.md [hunk 7]
```diff
@@ -1 +1,102 @@
...
+        # list of view functions and expected results
+        getMyParameter: *MY_PARAMETER
+        getFoo: *FOO
+      ozAcl:
+        *DEFAULT_ADMIN_ROLE : [*adminMultisig]
+```
+
+### ABIs
+
+All requried ABIs must be located in the same directory as the config and placed under `abi` folder. See [configs](/configs/).
```
Comment (TheDZhon) [inline README.md:93]: Shall we retrieve abis from Etherscan (even though it would provoke a bit of complication for proxies)?

File: README.md [hunk 8]
```diff
@@ -1 +1,102 @@
+<div>
+    <img alt="state-mate" src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Flidofinance%2Fstate-mate%2Fmain%2Fpackage.json&query=%24.version&label=state-mate&labelColor=white&color=green"/>
+    <img alt="Node.js" src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Flidofinance%2Fstate-mate%2Fmain%2Fpackage.json&query=%24.engines.node&style=flat&label=node.js&labelColor=rgb(62%2C%20109%2C%2026)&color=white"/>
+    <img alt="GitHub license" src="https://img.shields.io/github/license/lidofinance/state-mate?labelColor=orange&color=white"/>
+</div>
+
 # state-mate
+
```
Comment (TheDZhon) [inline README.md:8]: I miss a catchy project picture :cry:

**Review by TheDZhon — COMMENTED** [review]
:star2:

File: README.md (hunk 6 above)
Comment (mymphe) [reply README.md:28]: Technically any yarn will work, so I removed the version

File: README.md (hunk 7 above)
Comment (mymphe) [reply README.md:97]: For now, there is no such feature, so the doc reflects it. We'll update this section once it's delivered

Comment (TheDZhon) [reply README.md:97]: Agree, filed an issue #8

**Review by TheDZhon — APPROVED** [review]
:guitar:

**Pull request merged**